Cargo.toml

```diff
@@ -34,7 +34,7 @@ bson-u2i = ["bson/u2i"]
 async-trait = "0.1.42"
 base64 = "0.13.0"
 bitflags = "1.1.0"
-bson = "1.2.0"
+bson = { git = "https://github.com/mongodb/bson-rust" }
```

Comment:
Before we release the next alpha or beta on crates.io, we'll need to release and alpha/beta of `bson` to depend on here. In the meantime, just tracking the git repo should be fine.